bin/install

```diff
@@ -94,6 +94,18 @@ get_java() {
 			signature=${corretto_base}.sig
 			sigkey="A122542AB04F24E3"
         ;;
+    corretto-11.0.13.8.1)
+            local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
+			url=${corretto_base}
+			signature=${corretto_base}.sig
+			sigkey="A122542AB04F24E3"
+    ;;
+    corretto-17.0.1.12.1)
+            local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
+			url=${corretto_base}
+			signature=${corretto_base}.sig
+			sigkey="A122542AB04F24E3"
+    ;;
```

Comment:
```suggestion
        corretto-11.0.13.8.1)
            local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
			url=${corretto_base}
			signature=${corretto_base}.sig
			sigkey="A122542AB04F24E3"
    	;;
    	corretto-17.0.1.12.1)
            	local corretto_base="https://corretto.aws/downloads/resources/$(cut -d - -f2 <<< "$version")/amazon-$version-linux-x64.tar.gz"
			url=${corretto_base}
			signature=${corretto_base}.sig
			sigkey="A122542AB04F24E3"
    	;;
```

Reply:
@BaCaRoZzo I've fixed the indentation for the previous 2 sections. Now all branches have the same indentation. Wdyt?

Reply:
Hm. Something wrong with IDE... Fixing

Reply:
@BaCaRoZzo Fixed.